### DISCUSSION TITLE
Feature Request: Create an adapter for expo gl-view

### DISCUSSION THREAD

**Discussion by sergeymorkovkin** [pixijs/pixijs#9436] [General]
### Description

@bigtimebuddy Not sure why all initiatives got abandoned, but I personally believe this was a great idea to run PIXI inside a React Native app. Expo perfectly supports WebGL with their gl-view, while reanimated allows to run JS on the UI thread. So, why not? What is required for this?

**Comment by bigtimebuddy** [comment]
Expo adapter was not really on our radar, but this would probably be a good candidate for a community plugin.

A great starting place is to look at the node.js bundle http://github.com/pixijs/node which has an example of of a custom adapter implementation.
